phac-nml/irida-next
# Workflow table pagination

## What does this PR do and why?
Made the following changes to the workflow table:
- Added pagination
- Added new Job ID column
- Added sorting to all columns except `actions`

Fixes #470.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/9a993845-d170-4782-9468-56588c177cf0)

## How to set up and validate locally
1. Login to irida-next
2. Create some new workflows.
3. Navigate to the workflow page.
4. Verify table pagination & sorting work.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
